### PR TITLE
nf-test Transition CI

### PR DESCRIPTION
**Pull request by Emiller88**
This will allow us to migrate tests to nf-test at our own pace. We can leave an overlap as well like I've shown currently, so tests can be written piecemeal, without us needed to redo everything all at once.

This will be ready once https://github.com/askimed/nf-test/pull/80 is released 🚀

### CODE REVIEW

**Comment by lukfor** [comment]
We just released [v0.7.3](https://github.com/askimed/nf-test/releases/tag/v0.7.3) 🚀

**Comment by maxulysse** [comment]
Can you add some docs about how to create the files for the tests until we make it into tools?

File: tests/modules/nf-core/fastqc/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,32 @@
+nextflow_process {
+
+    name "Test Process FASTQC"
+    script "modules/nf-core/fastqc/main.nf"
+    process "FASTQC"
+    tag "fastqc"
+
+    test("Should run without failures") {
```
Comment (sateeshperi) [inline tests/modules/nf-core/fastqc/main.nf.test:8]:
```suggestion
    test("test_fastqc_single_end") {
```

**Review by sateeshperi — APPROVED** [review]
minor suggestion...otherwise lgtm

File: tests/modules/nf-core/fastqc/main.nf.test (hunk 1 above)
Comment (Emiller88) [reply tests/modules/nf-core/fastqc/main.nf.test:8]: Good catch, but I think we can do away with the _'s now. Thoughts? I had it actually just as "Single End", I guess I regenerated it?

Comment (sateeshperi) [reply tests/modules/nf-core/fastqc/main.nf.test:8]: yup...that works too

**Pull request merged**